frontend/src/components/personal/EmailChangeForm.tsx

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable react/jsx-no-bind */
```

Comment:
Please do not disable eslint rules

Reply:
corrected this by using useCallback in arrow function, although not sure its the best way